hypex/dataset/dataset.py

```diff
@@ -215,19 +224,24 @@ def agg(self, func: Union[str, List]):
         return self._convert_data_after_agg(self._backend.agg(func))
 
 
-class ExperimentData(Dataset):
+class ExperimentData:
     def __init__(self, data: Dataset):
-        self.additional_fields = Dataset._create_empty(index=data.index)
-        self.stats = Dataset._create_empty(index=data.columns)
-        self.additional_fields = Dataset._create_empty(index=data.index)
+        self._data = data
```

Comment:
why data? we discussed that we may have an incorrect interpretation of this field